crates/sui-graphql-rpc/src/context_data/db_data_provider.rs

```diff
@@ -260,87 +256,6 @@ impl PgManager {
         .await
     }
 
-    async fn multi_get_txs(
-        &self,
-        first: Option<u64>,
-        after: Option<String>,
-        last: Option<u64>,
-        before: Option<String>,
-        filter: Option<TransactionBlockFilter>,
-    ) -> Result<Option<(Vec<StoredTransaction>, bool)>, Error> {
-        let limit = self.validate_page_limit(first, last)?;
-        let descending_order = last.is_some();
-        let cursor = after
-            .or(before)
-            .map(|cursor| self.parse_tx_cursor(&cursor))
-            .transpose()?;
-
-        let mut after_tx_seq_num: Option<i64> = None;
-        let mut before_tx_seq_num: Option<i64> = None;
-        if let Some(filter) = &filter {
```

Comment:
This PR does not preserve this technique of translating checkpoint bounds into tx sequence number bounds.  I add it back in #15699, while maintaining a single DB roundtrip for the qeuery. cc @wlmyng